src/Components/CriticalCareRecording/HemodynamicParameters/CriticalCare__HemodynamicParameters.res

```diff
@@ -0,0 +1,133 @@
+let str = React.string
+
+let handleSubmit = (handleDone, state) => {
+	handleDone(state)
+}
+
+type action =
+|	SetBp_systolic(string)
+| SetBp_diastolic(string)
+|	SetPulse(string)
+|	SetTemperature(string)
+|	SetRespiratory_rate(string)
+|	SetRhythm(HemodynamicParametersType.rhythmVar)
```

Comment:
this will be coming from the individual Type file (see the other comments) check the file `CriticaCare__Types.res`

Reply:
done